lib/metrics/StatsClient.js

```diff
@@ -105,6 +129,37 @@ class StatsClient {
         return this._redis.incrEx(key, this._expiry, callback);
     }
 
+    /**
+     * Handles getting a list of global keys.
+     * @param {array} ids - Service identifiers
+     * @param {object} log - Werelogs request logger
+     * @param {function} cb - Callback
+     * @return {undefined}
+     */
+    getAllGlobalStats(ids, log, cb) {
+        const reqsKeys = ids.map(key => (['get', key]));
+        return this._redis.batch(reqsKeys, (err, res) => {
+            const statsRes = { requests: 0 };
+            if (err) {
+                log.error('error getting metrics', {
+                    error: err,
+                    method: 'StatsClient.getAllGlobalStats',
+                });
+                return cb(null, statsRes);
+            }
+            statsRes.requests = res.reduce((sum, curr) => {
+                if (curr[0]) {
+                    log.error('error getting metrics', {
```

Comment:
i'm wondering if this could potentially be more confusing to a user. In a global metrics case, the only time this would error would be if the pending metrics for a given site does not exist yet. So either a brand new location, or maybe somehow the metrics key was deleted. Or the new location was setup for replication and never was replicated to yet. These errors may show up frequently as cloudserver will query metrics often.

Maybe best to ignore these logs

Reply:
If the global key has not been created, the response will be `[ null, null ]` for each response of the batched request. I was thinking it would be helpful to log any error that could occur in an individual request from the batch. 

I've updated the variable names and added a comment to make this more explicit. Does this work for you, or do you think it's still best to omit these error logs?